McEngine/src/GUI/CBaseUIElement.h

```diff
@@ -1,112 +1,141 @@
-//================ Copyright (c) 2013, PG, All rights reserved. =================//
-//
-// Purpose:		the base class for all UI Elements
-//
-// $NoKeywords: $buie
-//===============================================================================//
-
-#ifndef CBASEUIELEMENT_H
-#define CBASEUIELEMENT_H
-
-#include "cbase.h"
-#include "Engine.h"
```

Comment:
(Don't ask me why I added Engine.h as an include to a header file here, I'll remove it in the future)